### ISSUE TITLE
Using this amd64 image fails on mac m1 arm64

### ISSUE THREAD

**Issue by utkarsh5474** [geerlingguy/docker-centos7-ansible#20]
I have been using this image from dockerhub ( https://hub.docker.com/r/geerlingguy/docker-centos7-ansible ) extensively for molecule testing. 
I recently upgraded to an m1 with arm64. 
Using the same image with rest of the things kept same as before, I hit multiple issues.
One of them was
`"stderr": "qemu: uncaught target signal 6 (Aborted) - core dumped",
    "stderr_lines": [
        "qemu: uncaught target signal 6 (Aborted) - core dumped"
    ],
    "stdout": "[0.019s][warning][os,container] Duplicate cpuset controllers detected. Picking /sys/fs/cgroup/cpuset, skipping /sys/fs/cgroup/cpuset.\n#\n# A fatal error has been detected by the Java Runtime Environment:\n#\n#  SIGSEGV (0xb) at pc=0x000000401e955282, pid=41174, tid=41211\n#\n# JRE version: OpenJDK Runtime Environment (Red_Hat-11.0.16.0.8-1.el7_9) (11.0.16+8) (build 11.0.16+8-LTS)\n# Java VM: OpenJDK 64-Bit Server VM (Red_Hat-11.0.16.0.8-1.el7_9) (11.0.16+8-LTS, mixed mode, sharing, tiered, compressed oops, g1 gc, linux-amd64)\n# Problematic frame:\n# J 207 c1 java.util.HashMap.afterNodeInsertion(Z)V java.base@11.0.16 (1 bytes) @ 0x000000401e955282 [0x000000401e955240+0x0000000000000042]\n#\n# No core dump will be written. Core dumps have been disabled. To enable core dumping, try \"ulimit -c unlimited\" before starting Java again\n#\n# An error report file with more information is saved as:\n# //hs_err_pid41174.log\nCompiled method (c1)    1368   88       3       java.util.HashMap::putVal (300 bytes)\n total in heap  [0x000000401e921690,0x000000401e923040] = 6576\n relocation     [0x000000401e921808,0x000000401e921948] = 320\n main code      [0x000000401e921960,0x000000401e922b00] = 4512\n stub code      [0x000000401e922b00,0x000000401e922bd0] = 208\n metadata       [0x000000401e922bd0,0x000000401e922bd8] = 8\n scopes data    [0x000000401e922bd8,0x000000401e922df0] = 536\n scopes pcs     [0x000000401e922df0,0x000000401e922fd0] = 480\n dependencies   [0x000000401e922fd0,0x000000401e922fd8] = 8\n nul chk table  [0x000000401e922fd8,0x000000401e923040] = 104\nCompiled method (c1)    1372   88       3       java.util.HashMap::putVal (300 bytes)\n total in heap  [0x000000401e921690,0x000000401e923040] = 6576\n relocation     [0x000000401e921808,0x000000401e921948] = 320\n main code      [0x000000401e921960,0x000000401e922b00] = 4512\n stub code      [0x000000401e922b00,0x000000401e922bd0] = 208\n metadata       [0x000000401e922bd0,0x000000401e922bd8] = 8\n scopes data    [0x000000401e922bd8,0x000000401e922df0] = 536\n scopes pcs     [0x000000401e922df0,0x000000401e922fd0] = 480\n dependencies   [0x000000401e922fd0,0x000000401e922fd8] = 8\n nul chk table  [0x000000401e922fd8,0x000000401e923040] = 104\nCould not load hsdis-amd64.so; library not loadable; PrintAssembly is disabled\n#\n# If you would like to submit a bug report, please visit:\n#   https://bugzilla.redhat.com/enter_bug.cgi?product=Red%20Hat%20Enterprise%20Linux%207&component=java-11-openjdk\n#"`

Having arm64 architecture support for this image (and the other docker ansible images) will be very useful. It'll bypass qemu so hopefully this issue will be gone too.

**Comment by geerlingguy** [comment]
Yeah; see my last commit: https://github.com/geerlingguy/docker-centos7-ansible/commit/83a66189e7de0c0e630197889bf7db5f83679139

For some reason it's impossible to get the build going on ARM64 for CentOS 7 :( — or at least very challenging.

**Comment by utkarsh5474** [comment]
Aah ok, I saw a similar [commit](https://github.com/geerlingguy/docker-centos8-ansible/commit/19ddd952369083296ac499a6a43766cca5a0e71b) for centos8 image too.
I'll give it a try too. Thanks.
